scraping.py

```diff
@@ -0,0 +1,101 @@
+requests.packages.urllib3.disable_warnings()
+
+def validateSuccessfulResponse(resp):
+    if resp.status_code != 200:
+        # demisto.results(errorEntry("Got response status " + str(resp.status_code) + " when fetching event query"))
+        print "error!!!! " + resp.text
+        exit(0)
+
+
+def getIncidentEvents():
+    fortiIncID = "111"
+    max_events = "30"
+    extendedData = "true"
+
+    session=requests.session()
+
+    headers = {
+        'Upgrade-Insecure-Requests': '1',
+        'Content-Type': 'application/x-www-form-urlencoded',
+        'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
+        'Accept-Encoding': 'gzip, deflate, br',
+        'Accept-Language': 'en-US,en;q=0.9,he;q=0.8'
+    }
+    data = {
+      'loginHtml': 'loginHtml',
+      'loginHtml:username': USERNAME,
+      'loginHtml:password': PASSWORD,
+      'loginHtml:userDomain': 'Empty',
+      'loginHtml:loginBtn': 'Log In',
+      'javax.faces.ViewState': VIEW_STATE
+    }
+
+    response = session.post('https://10.0.1.248/phoenix/login-html.jsf', headers=headers, data=data, verify=False)
+    response = session.get('https://10.0.1.248/phoenix/rest/h5/report/triggerEvent?rawMsg='+fortiIncID)
+
+    headers = {
+        'Accept': 'application/json, text/plain, */*',
+        'Content-Type': 'application/json',
+        'Accept-Encoding': 'gzip, deflate, br'
+    }
+
+    try:
+        jsonRes = response.json()
+        if (type(jsonRes) is not list) or (len(jsonRes) <= 0) or (jsonRes[0].get('right', None) is None):
+            # demisto.results(errorEntry("Got response status " + str(resp.status_code) + " when fetching event query"))
+            raise "error!!!! " + response.text
+    except Exception as err:
+        # demisto.results(errorEntry("Got response status " + str(resp.status_code) + " when fetching event query"))
+        print "some error " + str(err)
```

Comment:
you are printing the errors and not using the commented error entries on purpose?